src/components/AssetInput/AssetInput.spec.tsx

```diff
@@ -0,0 +1,264 @@
+                        type: "image",
+                        extension: "JPG",
+                        src: "https://picsum.photos/200/300",
+                    },
+                ]}
+                actions={actions}
+            />,
+        );
+
+        cy.get(ASSET_SINGLE_INPUT_ID).should("contain", "2000");
+        cy.get(ASSET_SINGLE_INPUT_ID).should("contain", "Upload");
+        cy.get(ASSET_SINGLE_INPUT_ID).should("contain", "JPG");
+        cy.get(ASSET_SINGLE_INPUT_THUMBNAIL_ID).should("have.css", "backgroundColor");
+        cy.get(ASSET_SINGLE_INPUT_THUMBNAIL_IMAGE_ID).should("have.attr", "src", "https://picsum.photos/200/300");
+    });
+
+    it("renders asset input with audio", () => {
+        mount(
+            <AssetInput
+                size={AssetInputSize.Small}
+                assets={[
+                    {
+                        source: "library",
+                        sourceName: "Foobar",
+                        name: "foo",
+                        extension: "MP3",
+                        size: 2000,
+                        type: "audio",
+                    },
+                ]}
+                actions={actions}
+            />,
+        );
+
+        cy.get(ASSET_SINGLE_INPUT_THUMBNAIL_AUDIO_ID);
+        cy.get(ASSET_SINGLE_INPUT_ID).should("contain", "Foobar");
+        cy.get(ASSET_SINGLE_INPUT_ID).should("contain", "MP3");
+    });
+
+    it("without images, image url should not exist", () => {
+        mount(
+            <AssetInput
+                assets={[
+                    {
+                        source: "upload",
+                        name: "foo",
+                        size: 2000,
+                        type: "image",
+                        extension: "JPG",
+                        src: "https://picsum.photos/200/300",
+                    },
+                    {
+                        source: "upload",
+                        name: "foo",
+                        size: 2000,
+                        type: "image",
+                        extension: "JPG",
+                        src: "https://picsum.photos/200/300",
+                    },
+                ]}
+                onMultiAssetClick={() => alert("Multi Asset clicked")}
+                size={AssetInputSize.Small}
+                actions={[]}
+            />,
+        );
+
+        cy.get(ASSETS_IMAGE_ID).last().should("not.have.attr", "style");
+        cy.get(ASSETS_IMAGE_ID).last().should("have.css", "backgroundColor");
+    });
+
+    it("renders with images", () => {
+        mount(
+            <AssetInput
+                onMultiAssetClick={() => alert("Multi Asset clicked")}
+                numberOfLocations={12}
+                assets={EXAMPLE_IMAGES}
+                size={AssetInputSize.Small}
+                actions={[]}
+            />,
+        );
+
+        cy.get(MULTI_ASSET_PREVIEW_ID);
+    });
+
+    it("renders with mixed assets", () => {
```

Comment:
You do not actually test if the assets are mixed

Reply:
I don't?